prober/utils.go

```diff
@@ -37,8 +38,20 @@ func chooseProtocol(ctx context.Context, IPProtocol string, fallbackIPProtocol b
 		Name: "probe_ip_protocol",
 		Help: "Specifies whether probe ip protocol is IP4 or IP6",
 	})
+
+	probeIPAddr := prometheus.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
This IP could have high cardinality, it's not safe to expose it.

Reply:
Understood. I've removed that.